tests/server/test_bundle_service.py

```diff
@@ -0,0 +1,210 @@
+import copy
+
+from fastapi.testclient import TestClient
+from requests import Response
+
+from bundle_service.main import app
+
+client = TestClient(app)
+
+HEADERS = {"Authorization": "foo"}
+
+VALID_CLAIM = {
+    "resourceType": "Claim",
+    "status": "active",
+    "created": "2014-08-16",
+    "use": "claim",
+    "type": {
+        "coding": [
+            {
+                "system": "http://terminology.hl7.org/CodeSystem/claim-type",
+                "code": "oral",
+            }
+        ]
+    },
+    "patient": {"reference": "Patient/1"},
+}
+
+VALID_PATIENT = {
+    "resourceType": "Patient",
+    "identifier": [{"system": "https://example.org/my_id", "value": "test-foo"}],
+}
+
+VALID_REQUEST_BUNDLE = {
```

Comment:
Not sure how much of the data that we're going to be getting will be in FHIR bundle format. 
Would be very useful to add a more simple PUT for FHIR resource type X.

Reply:
Have created a grip RESTFUL endpoint that accepted 
`curl -X POST -H "Content-Type: applicationjson" -d '{"vertex": "../../aced-data/grip-aced-data/Observation.ndjson"}' http://localhost:8201/api/writer/test/bulk-load` 
style curl request that would bulk load a vertex file to grip. Adding this functionality in here would be very useful.